### ISSUE TITLE
[Docs]: Document Admin Settings

### ISSUE THREAD

**Issue by spark-c** [appsmithorg/appsmith#13804]
There's not much detail in what the admin settings are, and a user was blocked by lack of information.

Docs page for reference:

https://docs.appsmith.com/setup/instance-configuration/admin-settings

**Comment by jnikhila** [comment]
@danciaclara - We can update it as part of Rehaul Phase 2.

**Comment by jnikhila** [comment]
Merged into #16603
